### PR TITLE
Add a feature for room MotDs

### PR DESCRIPTION
**Pull request by whalemer**
trying to improve on the current system of using polls for this

### CODE REVIEW

File: rooms.js [hunk 1]
```diff
@@ -1190,14 +1191,17 @@ class BasicChatRoom extends BasicRoom {
...
+		if (this.motd) {
+			roomMessage += `<br />MotD: ${this.motd}`;
+		}
 		if (this.modchat) {
 			roomMessage += `<br />You must be rank ${this.modchat} or higher to talk right now.`;
 		}
 		if (this.modjoin) {
 			roomMessage += `<br />You must be roomvoice or have room management capabilities to join the room right now.`;
 		}
-		if (this.slowchat && user.can('mute', null, this)) {
```
[the remark this answers is not in the document]
Comment (whalemer) [reply rooms.js]: fair enough

**Comment by whalemer** [comment]
My intention for this was for simpler and/or more temporary messages. Many users will just skip over the roomintro of rooms they're familiar with, and it's a lot harder to get a RO to put something in your intro than a mod.

Regarding the name, I don't think I've ever seen anything where the motd had to strictly be a day. that being said, if you have a better name, I don't have any particular attachment to calling a motd.

**Comment by Zarel** [comment]
The roomintro can be changed as often or as rarely as you want.

I am vetoing the idea of having two roomintros just so you can change one frequently and one rarely.

**Comment by Zarel** [comment]
Change the top half rarely and the bottom half frequently, if you must. But the total height restriction applies.

**Comment by bumbadadabum** [comment]
I also don't think the current implementation really works well for that
purpose. This seems equivalent to just putting an extra <div> in the
roomintro which i feel makes it rather minor and pointless. If you want
people to be notified I think it's better to implement it similarly to
polls as its own thing that is repeated in the room occasionally.

**Comment by whalemer** [comment]
...sure

**Comment by Zarel** [comment]
I agree that the poll situation sucks, but that's not really something I can fix programmatically.

**Comment by Zarel** [comment]
One thing we could do is that if a non-roomowner starts a poll, it auto-stops in one hour or something?

**Pull request closed without merging**